src/main/java/org/opensearch/knn/indices/ModelDao.java

```diff
@@ -369,12 +361,12 @@ private ActionListener<IndexResponse> getUpdateModelMetadataListener(
 
         @SneakyThrows
```

Comment:
sneakythrows are not supposed to be used in src files. it is mainly for tests. please remove this.

Reply:
ack